src/software/jetson_nano/services/motor.cpp

```diff
@@ -838,9 +838,9 @@ void MotorService::writeToDriverOrDieTrying(uint8_t motor, uint8_t address, int3
     // the chip to clear any bad values we just wrote and crash so everything stops.
     reset_gpio_.setValue(GpioState::LOW);
     CHECK(read_value == value) << "Couldn't write " << value
-                               << " to the TMC6100 at address " << address
-                               << " at address " << static_cast<uint32_t>(address)
+                               << " to the TMC6100"
                                << " on motor " << static_cast<uint32_t>(motor)
```

Comment:
Lets use `MOTOR_NAMES` instead of their ID instead 

https://github.com/UBC-Thunderbots/Software/blob/dea10f841ad24ef8ac8da743416e4c9495f9e83b/src/software/jetson_nano/services/motor.h#L402-L404